docs/docs-content/enterprise-version/upgrade/upgrade-k8s/airgap.md

```diff
@@ -0,0 +1,432 @@
+---
+sidebar_label: "Airgap"
+title: "Upgrade Airgap Palette Installed with Kubernetes"
+description: "Learn how to upgrade self-hosted airgap Palette."
+icon: ""
+sidebar_position: 10
+tags: ["palette", "self-hosted", "airgap", "kubernetes", "upgrade"]
+keywords: ["self-hosted", "enterprise", "airgap", "kubernetes"]
+---
+
+This guide takes you through the process of upgrading a self-hosted airgap Palette instance installed on Kubernetes.
+
+:::warning
+
+Before upgrading Palette to a new major version, you must first update it to the latest minor version available. Refer
+to the [Supported Upgrade Paths](../upgrade.md#supported-upgrade-paths) section for details.
+
+If your setup includes a PCG, you must also
+[allow the PCG to upgrade automatically](../../../clusters/pcg/manage-pcg/pcg-upgrade.md) before each major or minor
+Palette upgrade.
+
+:::
```

Comment:
I mentioned in the upgrade page how this warning box is a bit tricky to read. I think we may have to decouple the two different operations to improve the readability.